Features/expand.feature

```diff
@@ -0,0 +1,91 @@
+# language: nl
+
+# Nederlandse API Strategie:
+# API-10 Resources ondersteunen “lazy” en “eager” laden van relaties
+# API-11 Gelinkte resources worden expliciet en selectief mee-geladen
+
+Functionaliteit: Automatisch laden van sub-resources
+  "eager loading" wordt mogelijk gemaakt voor de volgende sub-resources van een woonplaats:
+  - geometrie
+  - bronhouder
+
+  "eager loading" wordt mogelijk gemaakt voor de volgende sub-resources van een openbare ruimte:
+  - woonplaats
+
+  "eager loading" wordt mogelijk gemaakt voor de volgende sub-resource van een nummeraanduiding:
+  - openbare ruimte
+  - woonplaats
+  
+  "eager loading" wordt mogelijk gemaakt voor de volgende sub-resource van een adresseerbaar object:
+  - nummeraanduiding (hoofdadres)
+  - nummeraanduiding (nevenadressen)
+  
+  "eager loading" wordt mogelijk gemaakt voor de volgende sub-resource van een verblijfsobject:
```

Comment:
Is het duidelijk genoeg dat dit komt naast wat standaard is voor een adresseeerbaar object wat ook een vbo kan zijn?